src/components/Badge/Badge.driver.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { StylableUnidriverUtil, UniDriver } from 'wix-ui-test-utils/unidriver';
+import style from './Badge.st.css';
```

Comment:
I know it's not documented anywhere, but we're having some problems importing `*.st.css` files inside tests.
It would be better if you could, instead of using the `StylableUnidriverUtil` here to check if it has the styles,
use a `data-*` hook for the `priority`.
So for instance to check if `isPrimary` we'll just need to check if `[data-priority="primary"]` exists.
It will work better, cause then tests will not need to import stylable files, which sometimes causes problems.
It's something that we'll need to fix in general in the library.